provisioner/terraform/serve_internal_test.go

```diff
@@ -0,0 +1,99 @@
+package terraform
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_getAbsoluteBinaryPath(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		ctx context.Context
+	}
+	tests := []struct {
+		name                   string
+		args                   args
+		terraformVersion       string
+		expectedAbsoluteBinary string
```

Comment:
Suggestion (non-blocking): Since this isn't used, it could be removed.

Reply:
Done, used it from within the function.